drakrun/drakrun/draksetup.py

```diff
@@ -115,8 +128,11 @@ def install(storage_backend, disk_size, iso_path, zfs_tank_name, unattended_xml)
 
     sha256_hash = hashlib.sha256()
 
+    logging.info("Calculating hash of iso")
+    iso_file_size = os.stat(iso_path).st_size
+    block_size = 65536 * 1024
     with open(iso_path, "rb") as f:
-        for byte_block in iter(lambda: f.read(4096), b""):
+        for byte_block in tqdm(iter(lambda: f.read(block_size), b""), total=math.ceil(iso_file_size / block_size)):
```

Comment:
Please rewrite this to look similarly to the code in drakpdb, i.e.:
```
with tqdm:
    with open:
        ...
```

Also, reading 64MiB chunk seems unreasonable, I'd keep this value around 128K. Above this value, glibc will start to mmap and munmap memory and the performance will drop.

Reply:
Sure, will make these changes.